client/landing/stepper/declarative-flow/internals/steps-repository/launch-big-sky/index.tsx

```diff
@@ -122,19 +122,14 @@ const LaunchBigSky: Step = function () {
 
 	function LaunchingBigSky() {
 		return (
-			<div className="processing-step__container">
-				<div className="processing-step">
-					<h1 className="processing-step__progress-step">
-						{ __( 'Launching the AI Website Builder' ) }
-					</h1>
-					{ ! isError && <ProgressBar className="processing-step__progress-bar" /> }
-					{ isError && (
-						<p className="processing-step__error">
-							{ __( 'Something unexpected happened. Please go back and try again.' ) }
-						</p>
-					) }
-				</div>
-			</div>
+			<>
+				<StepperLoader title={ __( 'Launching the AI Website Builder' ) } />
```

Comment:
The loading bar will stay rendered when there is an error.

Reply:
Addressed!